swarm/fuse/swarmfs_test.go

```diff
@@ -212,17 +213,23 @@ type testAPI struct {
 }
 
 func (ta *testAPI) mountListAndUnmountEncrypted(t *testing.T) {
+	log.Info("Starting mountListAndUnmountEncrypted test")
```

Comment:
Looks like a little too much for info level logging

Reply:
I respectfully disagree. Info is for high level info on what is going on. Error is just errors, and Warn of course warnings. So the next natural level to get an overview of what's going on is Info. Our code uses too much Debug level which makes it impossible to look for anything, and should use more the Trace level

Reply:
I also think that we don't need such `log.Info` before/after every test. The problem is that FUSE test suite is not coded according to conventions - we have one huge `TestFUSE` test, which calls all these subtests, which are not even related. We should just have a number of top-level `TestXXX` tests, where you can see the name of the tests being run, when you call `go test -v`.

The fact that we have too verbose `log.Debug` calls is another problem.

I don't have a strong opinion whether to keep the `log.Info`s or remove them right now, but we should probably refactor this whole test suite at some point.

Reply:
Furthermore it is very difficult to figure out which tests is deadlocking, I guess it is for this purpose that you added all these `log.Info`, so that it is visible in Travis? If this is the case, then I can totally understand, and we should leave them, until we have very high confidence that FUSE tests are not deadlocking.